src/components/ScreenWrapper/index.js

```diff
@@ -39,16 +39,31 @@ const ScreenWrapper = React.forwardRef(
             shouldDismissKeyboardBeforeClose,
             onEntryTransitionEnd,
             testID,
+            /**
```

Comment:
Our style recommends an empty line before these type of comments
```suggestion

            /**
```